src/main.rs

```diff
@@ -63,6 +63,8 @@ fn print_err_and_exit(err: errors::RustOrJsError) {
 }
 
 fn main() {
+  ansi_term::enable_ansi_support(); // For Windows 10
```

Comment:
nice! This was badly needed.

Reply:
But it crashes the build

Reply:
is it necessary or not? it seems like you've removed it.